.github/workflows/db_rollback.yml

```
@@ -0,0 +1,68 @@
+name: Rollback DB
+
+on:
+  workflow_dispatch:
+    inputs:
+      rollbackCount:
+        description: 'Number of migrations to rollback'
+        required: true
+        default: # of rollback you require in the file input
+      rollbackFile:
+        description: "File with migrations you want to rollback"
+        required: true
+        default: Filepath to yml
+
+jobs:
+  terraform-deploy:
+    name: Terraform Deploy
+    environment:
+      name: ${{ inputs.ENVIRONMENT }}
+    runs-on: ubuntu-latest
+    env:
+      ARM_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
+      ARM_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+      ARM_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      ARM_USE_OIDC: true
+    permissions:
+      id-token: write
+      contents: read
+    defaults:
+      run:
+        working-directory: ${{ inputs.TERRAFORM_DIRECTORY }}
+
+    outputs:
+      REGISTRY: ${{ steps.export-terraform-output.outputs.REGISTRY }}
+      APP: ${{ steps.export-terraform-output.outputs.APP }}
+
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Terraform Init
+        id: init
+        run: terraform init ${{ inputs.TERRAFORM_INIT_PARAMETERS }}
```

We don't need Terraform init parameters for our main environments.
```suggestion
        run: terraform init
```